src/server/modules/user/interfaces/TagManagementService.ts

```diff
@@ -0,0 +1,9 @@
+import { UserTagsQueryConditions } from '../../../repositories/types'
+
+interface TagManagementServiceInterface {
```

Comment:
slight note on naming here: should we name this `TagManagementService`, in line with both the file name, and how the interface for the similar URL management service is called `UrlManagementService` too? (and yet the repository interfaces are called `<Repository>Interface`... how confusing @.@)

Reply:
Can be named either IService or ServiceInterface.

Reply:
looks good for now, will note down for future review!